### PR TITLE
chore(*) backport envoy scripts changes

### PR DESCRIPTION
**Pull request by bartsmykla**
### Summary

To backport this changes I had to remove from the commits changes
related to bumping Envoy's version to 1.20

Backported changes:
- https://github.com/kumahq/kuma/commit/9a7dc1e5f1392f40364c913e407f4f564e054a11
- https://github.com/kumahq/kuma/commit/d7d64871fd5dc01cb71eaf9ee174f9547b2d4f9c
- https://github.com/kumahq/kuma/commit/a844addf978ec8b410f85b1114311b0a9966d587

### Full changelog

no changelog

### Issues resolved

no issues resolved

### Documentation

- [ ] Link to the website [documentation PR](https://github.com/kumahq/kuma-website/pull/XXX)

### Testing

- [ ] Unit tests
- [ ] E2E tests
- [ ] Manual testing on Universal
- [ ] Manual testing on Kubernetes

### Backwards compatibility

- [ ] Update [`UPGRADE.md`](UPGRADE.md) with any steps users will need to take
      when upgrading.
- [ ] Add `backport-to-stable` label if the code is backwards compatible. Otherwise, list breaking changes.

### CODE REVIEW

**Review by lahabana — APPROVED** [review]
LGTM but would love a check from @lobkovilya

**Review by jakubdyszkiewicz — APPROVED** [review]

**Review by lobkovilya — APPROVED** [review]

**Comment by lobkovilya** [comment]
wait, something wrong with CI

File: tools/releases/dockerfiles/Dockerfile.kuma-dp [hunk 1]
```diff
@@ -1,7 +1,9 @@
 FROM frolvlad/alpine-glibc:alpine-3.14_glibc-2.33
 
+ARG ENVOY_VERSION
```
Comment (lobkovilya) [inline tools/releases/dockerfiles/Dockerfile.kuma-dp:3]: it seems like ENVOY_VERSION is not set on CI

Comment (bartsmykla) [reply tools/releases/dockerfiles/Dockerfile.kuma-dp:3]: on it

Comment (lobkovilya) [reply tools/releases/dockerfiles/Dockerfile.kuma-dp:3]: it should be v1.18.4 not just 1.18.4

Comment (lobkovilya) [reply tools/releases/dockerfiles/Dockerfile.kuma-dp:3]: in `envoy.mk`

File: mk/envoy.mk [hunk 2]
```diff
@@ -1,32 +1,52 @@
-ENVOY_TAG ?= v1.18.4
-ENVOY_COMMIT_HASH ?=
+KUMA_DIR ?= .
+
+BUILD_ENVOY_FROM_SOURCES ?= false
+
+ENVOY_TAG ?= 1.18.4 # commit hash or git tag
```
Comment (lobkovilya) [inline mk/envoy.mk:5]:
```suggestion
ENVOY_TAG ?= v1.18.4 # commit hash or git tag
```

File: tools/releases/dockerfiles/Dockerfile.kuma-dp (hunk 1 above)
Comment (bartsmykla) [reply tools/releases/dockerfiles/Dockerfile.kuma-dp:3]: fixed, thank you for catching this!

**Pull request merged**